src/views/partials/header.ejs

```diff
@@ -98,11 +99,19 @@
         <div class="govuk-phase-banner">
             <p class="govuk-phase-banner__content">
                 <strong class="govuk-tag govuk-phase-banner__content__tag">
-                    <%= t('beta') %>
+                    <%= t('header.beta') %>
                 </strong>
                 <span class="govuk-phase-banner__text">
-                    <%= t('feedback') %>
+                    <%- t('header.feedback') %>
                 </span>
             </p>
         </div>
-    </div>
\ No newline at end of file
+    </div>
+
+    <% if (locals.developerPage) { %>
+    <div class="govuk-width-container app-width-container">
+        <div class="govuk-phase-banner">
+            <p class="govuk-phase-banner__content"><%- t('developer_warning')%></p>
+        </div>
+    </div>
+    <% } %>
```

Comment:
I wouldn't even bother making this conditional, this will be around for years and we want it on every page I would imagine?

Reply:
Sorry I was getting it confused with the existing gov phase banner for the service which looks like it's already there.

In that case it feels a bit excessive to call certain pages when literally the entire service is still in active development and subject to change, but I'm not willing to die on that hill if people want it 😄